arch/xtensa/include/xtensa_timer.h

```diff
@@ -127,8 +127,12 @@
  * "-DXT_TICK_PER_SEC=<value>" where <value> is a suitable number.
  */
 #ifndef XT_TICK_PER_SEC
-#define XT_TICK_PER_SEC    CONFIG_SYS_CLOCK_TICKS_PER_SEC
-#endif
+#if CONFIG_TICKLESS_KERNEL
```

Comment:
Why do we need this definition twice in xtensa_timer.h and xtensa_rtos.h?

Reply:
As XT_TICK_PER_SEC was degined in both the files, So I modified in both. 
Actually ctensa_timer is included into xtensa_rtos based on XT_RTOS_TIMER_INT value.

Can you please recommend which place we should change?

Reply:
the xtensa headers are a bit of a mess, we should clean this up but I feel it falls outside the scope of this particular PR